sw/device/silicon_creator/manuf/base/BUILD

```diff
@@ -349,6 +353,7 @@ bool_flag(
 _FT_PROVISIONING_CMD_ARGS = """
   --elf={sram_ft_individualize}
   --bootstrap={ft_personalize}
+  --bootstrap={firmware}
```

Comment:
we only want to bootstrap one image, not two, don't we? the combined image should be:
- Slot A: Perso fw
- Slot B: ROM_EXT + Owner FW (or just ROM_EXT for now).

Reply:
When we do the 1st bootstrap, we just bootstrap perso fw; then when we do the 2nd bootstrap (after provisioning SECRET1 OTP partition) we bootstrap perso fw + ROM_EXT/Owner FW bundle; to save time loading big chunks of firmware into flash. That's why I sent a list of bootstrap items (perso fw only and the combined image) here,

Reply:
Wondering if we didn't bootstrap Owner FW at this point, how could we push Owner FW after that?
Maybe we should confirm if the SPI rescue works on this version of ROM_EXT.
Could you help confirm that internally @anthonychen1251?

Reply:
Ah right, I forgot we settled on this approach @anthonychen1251.

@tommychiu-github : there are two bootstrap operations that happen during perso because the flash scrambling keys are [provisioned into OTP SECRET1 partition](https://cs.opensource.google/opentitan/opentitan/+/master:sw/device/silicon_creator/manuf/base/ft_personalize.c;l=225?q=ft_personalize.c&ss=opentitan%2Fopentitan), and don't take effect until after a reset, at which point the contents of the flash will be garbage again if scrambling is turned on, so we bootstrap again to make it the rest of the way through perso. @anthonychen1251 approach is designed to be more efficient, since the first time around we know we need to bootstrap again, there is no need to bootstrap code that will never get used initially.

Reply:
@timothytrippel Thanks for the detailed explanation! I think @tommychiu-github was also wondering about the process for loading Owner FW onto flash if it's not done during perso. Given that the bootstrap isn't disabled in the bring-up SKU, flashing the Owner Software via bootstrap seems feasible. I'm not sure if the SPI interface support for the rescue protocol is ready though.

Reply:
Ah I see. If only Perso code is loaded during perso, and bootstrap was not disabled, then after perso, one can simply use the `opentitantool bootstrap ...` command to load a ROM_EXT + owner FW image.

SPI rescue is not supported yet.